lms/templates/header.html

```diff
@@ -1,3 +1,4 @@
 ## mako
+<%page expression_filter="h"/>
 <%namespace name='static' file='static_content.html'/>
 <%include file="${static.get_themed_template_path(relative_path='theme-header.html', default_path='navigation.html')}" />
```

Comment:
The latest linter will check includes. This one would need a disable violation pragma, but you should ensure that these will be safe.